### PR TITLE
UI for shopcarts

### PR DESCRIPTION
**Pull request by gauridhawan**
# Pull Request Template

## Checklist:

- [X] I have written test cases for this change, and my code passed the test
- [X] I have updated documentation to reflect this change
- [X] I have attached this PR to an issue/user story

### CODE REVIEW

File: service/static/js/rest_api.js [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+
+        ajax.fail(function(res){
+            flash_message(res.responseJSON.message)
+        });
+    });
+
+
+    // ****************************************
+    // Update a shopcart
+    // ****************************************
```
Comment (KHafner) [inline service/static/js/rest_api.js:107]: Should this be removed for now too?

**Review by KHafner — APPROVED** [review]

File: service/static/js/rest_api.js (hunk 1 above)
Comment (gauridhawan) [reply service/static/js/rest_api.js:107]: I've kept it so that I can refer to it when writing the code for shopcart items

**Review by nonococoleo — APPROVED** [review]
LGTM, thx!

**Comment by KHafner** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/b5c2987e-b3d1-4601-9f28-34419832137d?env_id=ibm:yp:us-south "nyu-shopcart-service-f20-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/c869c127-ceda-423d-80b8-f74ee52b7378 "nyu-shopcart-service-f20-pipeline") deployed [shopcarts](https://github.com/NYU-DevOps2020-shopcarts/shopcarts/releases/tag/deploy-ibm-yp-us-south-kfh226-nyu.edu-dev-20201115-051901 "deploy-ibm-yp-us-south-kfh226-nyu.edu-dev-20201115-051901") to [dev](https://console.bluemix.net/apps/4aa42608-55fd-40d6-b27b-d99aa18a0658?env_id=ibm:yp:us-south "ibm:yp:us-south:kfh226@nyu.edu:dev"), including commit 68951d125db9c925d802421289d02fd4bee64cd8

**Comment by KHafner** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/b5c2987e-b3d1-4601-9f28-34419832137d?env_id=ibm:yp:us-south "nyu-shopcart-service-f20-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/c869c127-ceda-423d-80b8-f74ee52b7378 "nyu-shopcart-service-f20-pipeline") deployed [shopcarts](https://github.com/NYU-DevOps2020-shopcarts/shopcarts/releases/tag/deploy-ibm-yp-us-south-kfh226-nyu.edu-prod-20201208-144028 "deploy-ibm-yp-us-south-kfh226-nyu.edu-prod-20201208-144028") to [prod](https://console.bluemix.net/apps/ac06a6b3-040d-43d7-b290-6bfbba3f8c41?env_id=ibm:yp:us-south "ibm:yp:us-south:kfh226@nyu.edu:prod"), including commit 68951d125db9c925d802421289d02fd4bee64cd8

**Pull request merged**